content/blog/2021-02-17-cml-runner-prerelease.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 'Pre-Release Notes: Automatically Train Models in the Cloud with CML'
+date: 2021-02-17
+description: |
+  New features are here to make launching cloud 
+  compute for continuous integration workflows 
+  shorter, sweeter and easier than ever. Plus, 
+  a new GitHub Action to setup CML means more 
+  ways to use CML without our Docker container.
+descriptionLong: |
+  New features are here to make launching cloud 
+  compute for continuous integration workflows 
+  shorter, sweeter and easier than ever. Plus, 
+  a new GitHub Action to setup CML means more 
+  ways to use CML without our Docker container.
+picture: 2021-02-17/gh_actions_art.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/cml-self-hosted-runners-on-demand-with-gpus/462
+tags:
+  - CML
+  - Continuous Machine Learning
+  - GitHub Actions
+  - GitLab
+  - Cloud
+---
+
+Today, we're pre-releasing some new features in Continuous Machine Learning, or
+[CML](https://cml.dev)- our open source project to adapt popular continuous
+integration (CI) systems like GitHub Actions and GitLab CI for data science.
+
+Here's what you'll get:
+
+## Brand new method to provision cloud compute for your CI workflows
+
+In the initial CML release, we provided a method user Docker Machine to launch
+instances on AWS, Azure and GCP from your GitHub Actions and GitLab CI
+workflows. We've completely redone this recipe using Terraform and now it's
+shorter, sweeter, and more powerful than ever!
+
+Check out this example workflow to launch an EC2 instance from a GitHub Action
+workflow and then train a model.
```

Comment:
Can we compare this with the old method like - there are only 50 lines of code instead of 70.

Reply:
Added, "In the first CML release, this took [more than 30 lines of code](https://github.com/iterative/cml_cloud_case/blob/master/.github/workflows/cml.yaml) to configure. Now it's just one function."